### ISSUE TITLE
Scrobbling from Android 8+

### ISSUE THREAD

**Issue by deved99** [krateng/maloja#375]
Hello!

I deployed Maloja in my home infrastructure, but I couldn't find a way to scrobble from my phone, which is running Android 13.

I tried some clients, but I had the following problems:
- In Simple Scrobbler, I had the same symptoms as described in [this issue](https://github.com/krateng/maloja/issues/69).
- In Pano Scrobbler, the issue seems to be that Maloja is talking in HTTP, not in HTTPS.

Since I own a server with a public IP, I configured a reverse proxy to the Maloja instance deployed in my home. This way I could install a certificate with Lets Encrypt. This works, but I'm a bit worried for the following reasons:
1. It feels like a flaky solution,
2. It's exposed publicly and I couldn't find a way to require a login, this feels unnecessary.

Is there a way to avoid using this unnecessary proxy?

**Comment by krateng** [comment]
Having Maloja behind a reverse proxy is the intended way to do it, there is no native support for SSL. Are you saying you just want to scrobble from inside your home network anyway?

**Comment by deved99** [comment]
Yes, I just want to scrobble from inside my home network. I have setup a VPN through which my phone is always connected to my home network.

The problem is that Panu Scrobbler requires HTTPS, but since everything is in a private network, I can't create a valid certificate.

Some months have passed and now I think I have 2 better solutions:
- Create a self-signed certificate and install it on my phone (I have to check if Panu Scrobbler supports that)
- Install Maloja on the remote server and avoid the unnecessary proxying.

Do you have any better solution?

**Comment by krateng** [comment]
If you can host your reverse proxy in your home network, I would just use a real certificate (DNS challenge), but not expose the proxy to the public at all. If your reverse proxy is outside, you could potentially filter by IP if your ISP doesn't change your IP; or add it to your VPN too.

**Comment by deved99** [comment]
My reverse proxy is outside, so my best option is, as you said, filter by IP the requests, allowing only the ones over VPN.
Thanks for your help!
